app.js

```diff
@@ -53,16 +56,20 @@ function addClass(insertedDiv, letter, i) {
 function gameSetup() {
   addLetterListeners();
   insertNextImg();
-  let rndPickedProverb = proverbs[getRandomInt(proverbs.length)];
-  sentence.innerHTML = encryptSentence(rndPickedProverb);
+  state.proverb = proverbs[getRandomInt(proverbs.length)].toLowerCase();
+  displayProgress();
 }
 
-function addLetterListeners() {
+function displayProgress(){
```

Comment:
you need to decide if `{` has a space before or not (there are different options in your code). 
The most beautiful is:
`function someName() {`